ruby/lib/ThreatExchange/version.rb

```diff
@@ -1,3 +1,3 @@
 module ThreatExchange
-  VERSION = "0.0.9"
+  VERSION = "0.0.10"
```

Comment:
wondering if we should update minor or even major instead, since the constructor signature changed